docs/tutorial/notebook/train_ss_model_using_pytorch.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p>This file needs an editing pass for language and organization. Also, rephrase from a user's perspective who is trying to learn what functionality is available and exactly what it will do. As an example, mention the inputs needed to make the main steps work (e.g. how do you get the dataset?) and what is the result. e.g. after running training, you should have a checkpoint (where?) that you can use for testing or inference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p><strong>To Do (Sanskar):</strong> editing pass for language and organization</p>

Reply:
Done